test/integration/multi-cluster-deployment/README.md

```diff
@@ -0,0 +1,174 @@
+# Kubestellar multi-cluster workload deployment with kubectl
+
+## Prerequisites
+See the list [here](https://github.com/kubestellar/kubestellar/blob/ks-0.20/test/pre-reqs.md).
+
+## Running the test using a script
+- Clone the repo and checkout the ks-0.20 branch
+```
+git clone git@github.com:kubestellar/kubestellar.git
+cd kubestellar
+git checkout -b ks-0.20 origin/ks-0.20
+```
+- Run the test
+```
+cd test/integration/multi-cluster-deployment
+./run-test.sh
+```
+
+## Running the test steps manually
+### Setup Kubestellar
+- Clone the repo and checkout the ks-0.20 branch
+```
+git clone git@github.com:kubestellar/kubestellar.git
+git branch -b ks-0.20 origin/ks-0.20
+cd kubestellar
+```
+
+Create a Kind hosting cluster with nginx ingress controller and KubeFlex operator.
+```
+kflex init --create-kind
+```
+
+Create an inventory & mailbox space of type vcluster running OCM (Open Cluster Management) directly in KubeFlex. Note that -p ocm runs a post-create hook on the vcluster control plane which installs OCM on it.
+```
+kflex create imbs1 --type vcluster -p ocm
+```
+
+Create a Workload Description Space wds1 directly in KubeFlex.
+```
+kflex create wds1
+kubectl config use-context kind-kubeflex
+kubectl label cp wds1 kflex.kubestellar.io/cptype=wds
+
+cd ../../../
+make ko-build
+make install-local-chart
+cd -
+```
+
+Create clusters and register with OCM.
+```
+kind create cluster --name cluster1
+kubectl config rename-context kind-cluster1 cluster1
+clusteradm --context imbs1 get token | grep '^clusteradm join' | sed "s/<cluster_name>/cluster1/" | awk '{print $0 " --context 'cluster1' --force-internal-endpoint-lookup"}' | sh
+
```

Comment:
For now this is good, but I think we should consider if making this a script or command in a makefile, after all the only thing that changes is the name of the cluster.